Source/Private/LevelStatsCollector.cpp

```diff
@@ -1,24 +1,65 @@
 ﻿#include "LevelStatsCollector.h"
 
+#include "LevelStatsCollectorState.h"
+
 #include <Components/SceneCaptureComponent2D.h>
+#include <Engine/Engine.h>
 #include <Engine/LevelBounds.h>
 #include <Engine/TextureRenderTarget2D.h>
 #include <ImageUtils.h>
 
+FCustomPerformanceChart::FCustomPerformanceChart( const FDateTime & in_start_time, const FString & in_chart_label, const FString & in_output_path ) :
+    FPerformanceTrackingChart( in_start_time, in_chart_label ),
+    CustomOutputPath( in_output_path )
+{
+}
+
+void FCustomPerformanceChart::DumpFPSChartToCustomLocation( const FString & in_map_name )
+{
+    TArray< const FPerformanceTrackingChart * > charts;
+    charts.Add( this );
+
+    DumpChartsToOutputLog( AccumulatedChartTime, charts, in_map_name );
+
+#if ALLOW_DEBUG_FILES
+    IFileManager::Get().MakeDirectory( *CustomOutputPath, true );
+
+    {
+        const auto log_filename = CustomOutputPath / CreateFileNameForChart( TEXT( "FPS" ), in_map_name, TEXT( ".log" ) );
+        DumpChartsToLogFile( AccumulatedChartTime, charts, in_map_name, log_filename );
+    }
+
+    {
+        const auto map_and_chart_label = ChartLabel.IsEmpty() ? in_map_name : ( ChartLabel + TEXT( "-" ) + in_map_name );
+        const auto html_filename = CustomOutputPath / CreateFileNameForChart( TEXT( "FPS" ),
+                                                          *( map_and_chart_label + TEXT( "-" ) + CaptureStartTime.ToString() ),
+                                                          TEXT( ".html" ) );
+        DumpChartsToHTML( AccumulatedChartTime, charts, map_and_chart_label, html_filename );
+    }
+#endif
+}
+
+FString FCustomPerformanceChart::CreateFileNameForChart( const FString & /* chart_type */, const FString & /* in_map_name */, const FString & file_extension )
```

Comment:
if some parameters are not used you can remove them

Reply:
You once told me to set unused parameter like i'm doing here. But done, still.

Reply:
for functions you can not change, like overridden functions. Here it's a function you created so you can just remove the parameter